support/dynamics-365/guides/application-functionality/cant-delete-guide.md

```diff
@@ -1,31 +1,37 @@
 ---
-title: Can't delete a guide
-description: Provides a resolution for an issue where you can't delete a guide because it's associated to another record.
+title: Can't delete a guide in Dynamics 365 Guides
+description: Provides a resolution for an issue where you can't delete a guide because it's associated with another record in Dynamics 365 Guides.
 ms.author: davepinch
 author: davepinch
 ms.reviewer: v-wendysmith, mhart
-ms.date: 11/02/2023
+ms.date: 11/08/2023
 ms.custom: bap-template
 ---
-# Can't delete a guide
+# Can't delete a guide in the Guides model-driven app
 
-This article provides a resolution for an issue where you can't delete a guide because it's associated with another record. Only an admin can delete a guide.
+This article provides a resolution for an issue where you can't delete a guide because it's associated with another record in Microsoft Dynamics 365 Guides.
 
 ## Symptoms
 
-When trying to delete a guide in the Guides model-driven app, the error "The record cannot be deleted because it is associated with another record" displays.
+When you try to delete a guide in the [Guides model-driven app](/dynamics365/mixed-reality/guides/model-driven-app-overview), you receive the following error message:
+
+> The record cannot be deleted because it is associated with another record.
 
 ## Resolution
 
-1. In the [Guides model-driven app,](/dynamics365/mixed-reality/guides/model-driven-app-overview) open the guide you're trying to delete.
+> [!NOTE]
+> You need administrator permissions to delete a guide.
+
+To delete a guide, follow these steps:
 
-1. Select the **Related** tab and then **Guide Operator Sessions**.
+1. In the Guides model-driven app, open the guide you try to delete.
```

Comment:
Shouldn't this be "open the guide you tried to delete"? Or you're trying to delete?

Reply:
Hi @v-wendysmith, I think your version is better. Here's the suggestion from Bing Chat and I updated back to "you're trying to delete". Thanks for the double-check!

Reply:
Thanks Lucy!